README.rst

```diff
@@ -222,7 +283,7 @@ for).
 
 You can easily generate a prediction following these steps::
 
-    $api = new BigML\BigML("myusername", "my_api_key");
+    $api = new BigML\BigML();
```

Comment:
Once your credentials have been properly stored as environment variables (see [the section]), you can easily...